tests/rkt_api_service_test.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build coreos
```

Comment:
I think we could loose the constraint a bit here, `kvm` is going to use a lot of those tests too.

Maybe instead of whitelisting, we should rather blacklist, i.e. `// +build !fly` in this case?